initramfs/scripts/local-premount/resize

```diff
@@ -103,11 +103,12 @@ if [ "$min_free_space" -lt "$free_space" ]; then
         gpt)
             # do_gpt needs the device name
             do_gpt "$device" >>$LOGFILE 2>&1
+            resizeopts="-p"
```

Comment:
Can you please add a flag or test ensuring that `resizeopts` is never empty so that this bug doesn't come back with some refactoring later on. Please include a comment describing why that is needed.